wpeverest/everest-forms
# Fix - Perfect scrollbar in RTL

### All Submissions:






### Changes proposed in this Pull Request:
This PR fixes the following issue with RTL:
> Fields List and Field Options containers' perfect scrollbar not working properly. If you keep scrolling and then switch between them, all the contents disappear.

### How to test the changes in this Pull Request:

1. Switch to a RTL language
2. Go to an EVF form
3. See if the scrollbars are working properly in the fields list and field options tabs

### Types of changes:




### Other information:




### Changelog entry

> Fix - Perfect scrollbar in RTL
